src/base/system_parameter.c

```diff
@@ -6228,7 +6250,44 @@ SYSPRM_PARAM prm_Def[] = {
    (void *) NULL, (void *) NULL,
    (char *) NULL,
    (DUP_PRM_FUNC) NULL,
-   (DUP_PRM_FUNC) NULL}
+   (DUP_PRM_FUNC) NULL},
+#if defined(SUPPORT_DEDUPLICATE_KEY_MODE)
+  {PRM_ID_USE_DEDUPLICATE_KEY_MODE_OID_TEST,
+   PRM_NAME_USE_DEDUPLICATE_KEY_MODE_OID_TEST,
+   (PRM_FOR_CLIENT | PRM_HIDDEN),
+   PRM_BOOLEAN,
+   &prm_use_deduplicate_key_mode_oid_flag,
+   (void *) &prm_use_deduplicate_key_mode_oid_default,
+   (void *) &PRM_USE_DEDUPLICATE_KEY_MODE_OID,
+   (void *) NULL, (void *) NULL,
+   (char *) NULL,
+   (DUP_PRM_FUNC) NULL,
+   (DUP_PRM_FUNC) NULL},
+  {PRM_ID_DEDUPLICATE_KEY_MODE,
+   PRM_NAME_DEDUPLICATE_KEY_MODE,
+   (PRM_FOR_CLIENT | PRM_USER_CHANGE | PRM_FOR_SESSION),
+   PRM_KEYWORD,
```

Comment:
PRM_BOOLEAN?

please refer to the `boolean_words` at line 6259.

Reply:
PRM_ID_USE_DEDUPLICATE_KEY_MODE_OID_TEST is PRM_BOOLEAN
PRM_ID_DEDUPLICATE_KEY_MODE                          is PRM_KEYWORD.

I'm not sure exactly what you're talking about.

Reply:
PRM_BOOLEAN also accepts "on" and "off". It seems that this parameter does not have to be the keyword type parameter.

Reply:
It seems out of character because I wouldn't accept forms like "deduplicate_key yes" "deduplicate_key 1".
It would be weird if it looked something like "deduplicate_key 1(16)"

Reply:
Because it is a system parameter, not a SQL syntax, So I thought we don't need to define the same key values. It is up to you.

Reply:
I just thought so too.
We will reflect your opinion.
Thank you.